src/server/routes/consents.ts

```diff
@@ -279,9 +294,23 @@ router.get(
       const { read, pageTitle: _pageTitle } = consentPages[pageIndex];
       pageTitle = _pageTitle;
 
+      // TODO: decide which approach to use
+      const newslettersTestActive =
+        state.abTesting.participations[ONBOARDING_NEWSLETTERS_TEST_ID]
+          ?.variant === ONBOARD_NEWSLETTERS_TEST_VARIANT;
+      // const newslettersTestActive = state.abTestAPI.isUserInVariant(
+      //   ONBOARDING_NEWSLETTERS_TEST_ID,
+      //   ONBOARD_NEWSLETTERS_TEST_VARIANT,
+      // );
```

Comment:
in general this is the approach that we've taken in the past

Reply:
Okay, will stick with that then.

Reply:
(discussed away from this PR with Mahesh)

For some reason `abTestAPI` was not available during cypress runs, so have switched to the other approach to unblock this feature.